Configuration/TypoScript/Plugins/kitodo.typoscript

```diff
@@ -188,7 +188,8 @@ plugin.tx_dlf_imagemanipulationtool {
 plugin.tx_dlf_pdfdownloadtool < plugin.tx_dlf_toolbox
 plugin.tx_dlf_pdfdownloadtool {
     settings {
-        tools = pdfdownloadtool
+        fileGrpsImageDownload = DEFAULT
+        tools = tx_dlf_pdfdownloadtool,imagedownloadtool,fulltextdownloadtool
```

Comment:
Could you use unified naming for tools - `tx_dlf_xyztool` or `xyztool`?

Reply:
I prefer the shorter version but feel free to choose whatever fits you better :)